app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -105,3 +114,27 @@ function RowExitWalletItem (): JSX.Element {
     </TouchableOpacity>
   )
 }
+
+function ViewRecoveryWords ({ onPress }: { onPress: () => void}): JSX.Element {
+  return (
+    <TouchableOpacity
+      testID='view_recovery_words'
+      style={tailwind('bg-white p-4 flex-row items-center justify-between border-b border-gray-200')}
+      onPress={onPress}
+    >
+      <Text style={tailwind('font-medium')}>
+        {translate('screens/Settings', 'Recovery Words')}
+      </Text>
+      <MaterialIcons
+        name='chevron-right'
+        style={[tailwind('text-black')]}
+        size={24}
+      />
+    </TouchableOpacity>
+  )
+}
+
+function getRecoveryWords (): string[] {
```

Comment:
turn this into a Promise<string[]> since the implementation will definitely be a promise

Reply:
updated